watchers/watcher_test.go

```diff
@@ -0,0 +1,317 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package watchers_test
+
+import (
+	"context"
+	"fmt"
+
+	logger "github.com/DataDog/chaos-controller/log"
+	"github.com/DataDog/chaos-controller/watchers"
+	"github.com/DataDog/chaos-controller/watchers/mocks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/stretchr/testify/mock"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+	k8scache "sigs.k8s.io/controller-runtime/pkg/cache"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+var _ = Describe("watcher", func() {
+	const watcherName = "watcher-name"
+
+	var (
+		watcher          watchers.Watcher
+		cacheMock        mocks.CacheMock
+		informerMock     mocks.InformerMock
+		handlerMock      mocks.HandlerMock
+		targetObjectType client.Object
+		err              error
+		cacheContextFunc func() (ctx context.Context, cancel context.CancelFunc)
+	)
+
+	// Arrange
+	namespaceName := types.NamespacedName{Name: "namespace-name", Namespace: "namespace"}
```

Comment:
absolutely ridiculous nitpick from me, but using "namespace-name" as the name in one test, but as the namespace in another test is confusing

Reply:
Indeed, it's often a good idea to be consistent 👍

Reply:
Unresolving because they're still inconsistent